bin/common/password.py

```diff
@@ -169,18 +175,19 @@ def init_mysql_passwords():
     get_mysql_uat_password()
     get_mysql_production_password()
 
-def init_all_passwords():
-    '''
-    Ask the user for all passwords used by syco, and add to passwordstore.
 
-    '''
-    get_root_password()
-    get_svn_password()
+def init_all_passwords():
+    """Ask the user for all passwords used by syco, and add to passwordstore."""
+    get_ca_password()
+    get_glassfish_admin_password()
+    get_glassfish_master_password()
     get_ldap_admin_password()
     get_ldap_sssd_password()
-    get_glassfish_master_password()
-    get_glassfish_admin_password()
-    get_user_password("glassfish")
-    get_switch_icmp_password()
+    get_master_password()
     get_redis_production_password()
+    get_root_password()
+    get_root_password_hash()
+    get_svn_password()
```

Comment:
You are right. get_svn_password is not needed any more. (It was a reorder of the function calls)